### DISCUSSION TITLE
direction vertical on cards effect

### DISCUSSION THREAD

**Discussion by kilianso** [nolimits4web/swiper#5149] [Q&A]
Hey there! Am i right, that direction: "vertical" is currently not supported when using effect: "cards"?

Trying to achieve a horizontal cardstack on Desktop while having a vertical one on Mobile. Any suggestions?

**Comment by kaleidografik** [comment]
Looking for the same, seems to adjust the stacking positions, but the slider itself breaks?

**Comment by marcoosvlopes** [comment]
It is, but you need to specify a height for the swiper in vertical mode, or it will try to fill the container (the hole screen) and will not work as expected. Try something like this:

```
.swiper {
  height: 400px;
}
```
